### DISCUSSION TITLE
Different users with different profiles.

### DISCUSSION THREAD

**Discussion by jsantosdias** [rffrasca/PDFKeeper#34] [Q&A] [answered]
Hello!
Is there a way of usking PDFKeeper with different users with different profiles (like ability or not to add or delete files...).
Thank you!

**Comment by rffrasca** [comment, marked as answer]
PDFKeeper is only compatible with SQLite and Oracle Database. With Oracle, you can with user accounts and grants, however, with SQLite, this is not possible.
